azure/templates/run-tests.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  - name: smoke_tests
+    type: boolean
+    displayName: Toggle smoke tests
+    default: false
+  - name: integration_tests
+    type: boolean
+    displayName: Toggle integration tests
+    default: false
+
+steps:
+
+  - bash: poetry install
+    workingDirectory: $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)
+    displayName: Setup tests
+
+  - template: "azure/components/aws-assume-role.yml@common"
+    parameters:
+     role: "auto-ops"
+     profile: "apm_ptl"
+
+  - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
+    parameters:
+     secret_file_ids:
+       - ptl/app-credentials/jwt_testing/non-prod/JWT_TESTING_WITH_ASID_PRIVATE_KEY
+       - ptl/app-credentials/jwt_testing/non-prod/JWT_TESTING_PRIVATE_KEY
+       - ptl/app-credentials/jwt_testing/non-prod/ID_TOKEN_TESTING_PRIVATE_KEY
+     config_ids:
+       - /ptl/azure-devops/env-int/test-app/internal-testing-int/CLIENT_ID_INT
+       - /ptl/azure-devops/env-int/test-app/internal-testing-int/CLIENT_SECRET_INT
+     secret_ids:
+       - ptl/app-credentials/jwt_testing/non-prod/JWT_TESTING_API_KEY
```

Comment:
@acarriedev Are these available in all environments where smoke tests are, ie. if it was run for prod.

Reply:
I don't believe there's one for prod. It might be doable to create some though people don't usually run automated tests against prod.

Reply:
I don't think we want to run the tests in prod, we were just considering what effect it would have if it failed to find these files in some environments.